SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+quic-go still in development. This means that there may be problems in our protocols,
+or there may be mistakes in our implementations.
+We take security vulnerabilities very seriously. If you discover a security issue,
+please bring it to our attention right away!
+
+## Reporting a Vulnerability
+
+If you find a vulnerability that may affect live deployments -- for example, by
+exposing a remote execution exploit -- please **send your report privately** to
```

Comment:
Maybe enable GH private vuln reporting instead and link to that?

Reply:
Done and updated the PR.